README.md

```diff
@@ -190,8 +190,8 @@ async function renditionCallback(source, renditions, outdir, params) => {
 
 ### Worker Options (optional)
 Optional parameters to pass into workers
-- disableSourceDownload: Boolean used to disable the source download (defaults to false)
-- disableRenditionUpload: Boolean used to disable the rendition upload (defaults to false)
+- disableSourceDownload: Boolean used to disable the source download (defaults to false).
+- disableRenditionUpload: Boolean used to disable the rendition upload (defaults to false). Use this flag only if no generated rendition is expected as a processing result.
```

Comment:
I would add more of a warning that this will make the worker activation fail since the sdk expects a rendition output.

I would even add a dummy write to a rendition.path in the renditionCallback example below so they know even though the sdk won't upload a rendition, they still need to pass a dummy rendition in so the sdk doesn't fail